pages/docs/setup/connect.mdx

```diff
@@ -367,6 +367,14 @@ This view is helpful for debugging connection issues or verifying rolling deploy
 
 ![App worker observability](/assets/docs/connect/cloud-app-workers.png)
 
+## Syncing and Rollbacks
+
+When deploying a new version of your application, the first instance to reach `connect()` will kick off a sync during the connection setup process. This will update function configurations to the desired state configured in your code.
+
+`connect` supports rolling releases: During a deployment of your app, Inngest will run functions on all connected workers, regardless of the version, as long as they are able to process a request for a given function. This prevents traffic from concentrating on a single instance during rollouts and causing a thundering herd issue.
```

Comment:
When reading this my key question is what "version" are we talking about here? An "app version?" or a function version? function versions are opaque to the user so we may want to clarify this.

If I understand correctly, as a rolling release happens workers running old versions of my app might still handle function execution which seems more like a warning rather than a feature. We may want to phrase it like that. Not sure.